src/Dscp/DB/DSL/Interpret/Sqlite3.hs

```diff
@@ -0,0 +1,188 @@
+
+{-# LANGUAGE QuasiQuotes #-}
+
+module Dscp.DB.DSL.Interpret.Sqlite3 where
+
+import Universum
+
+import Control.Monad.Reader
+import Control.Monad.Trans
+
+import qualified Data.Set as Set (Set, singleton, member, empty)
+
+import Database.SQLite.Simple hiding (query)
+import Database.SQLite.Simple.ToField (ToField(..))
+import Database.SQLite.Simple.FromField (FromField(..))
+
+import Text.InterpolatedString.Perl6
+
+import Dscp.Core.Types ( SignedSubmission(..)
+                       , Submission(..)
+                       , SubmissionType(..)
+                       , SubmissionWitness(..)
+                       , SubmissionSig
+                       , Assignment(..)
+                       , AssignmentType(..)
+                       , Address(..)
+                       , CourseId(..)
+                       , Grade(..)
```

Comment:
This seems nice, but better let `stylish-haskell` deal with that (at least for sake of uniformity).